starport/interface/cli/starport/cmd/module-create.go

```diff
@@ -17,13 +46,41 @@ func NewModuleCreate() *cobra.Command {
 		Args:  cobra.MinimumNArgs(1),
 		RunE:  createModuleHandler,
 	}
+	c.Flags().Bool(flagIBC, false, "scaffold an IBC module")
+	c.Flags().String(flagIBCOrdering, "none", "channel ordering of the IBC module [none|ordered|unordered]")
 	return c
 }
 
 func createModuleHandler(cmd *cobra.Command, args []string) error {
+	var options []scaffolder.ModuleCreationOption
+
+	// Check if the module must be an IBC module
+	ibcModule, err := cmd.Flags().GetBool(flagIBC)
```

Comment:
It would be cleaner to define all the flags and commands to variables in the very beginning of `createModuleHandler` and use them below.